otter/convergence/planning.py

```diff
@@ -216,7 +220,7 @@ def drain_and_delete_a_server(server):
     delete_error_steps = (
         [DeleteServer(server_id=server.id) for server in servers_in_error] +
         [RemoveNodesFromCLB(lb_id=lb_node.description.lb_id,
-                            node_ids=(lb_node.node_id,))
+                            node_ids=pset([lb_node.node_id]))
```

Comment:
If the code previously accepted either, and that's a defect, there should probably be a typecheck for `PSet` in `RemoveNodesFromCLB`.